vajram/vajram-krystex/src/main/java/com/flipkart/krystal/vajramexecutor/krystex/SessionInputDecoratorConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.flipkart.krystal.vajramexecutor.krystex;
+
+import com.flipkart.krystal.krystex.decoration.LogicExecutionContext;
+import com.flipkart.krystal.krystex.decoration.MainLogicDecorator;
+import com.flipkart.krystal.krystex.decoration.MainLogicDecoratorConfig.DecoratorContext;
+import com.flipkart.krystal.vajram.Vajram;
+import com.flipkart.krystal.vajram.adaptors.DependencyInjectionAdaptor;
+import java.util.function.Function;
+
+public record SessionInputDecoratorConfig(
```

Comment:
We should reuse MainLogicDecoratorConfig

Reply:
Reusing MainLogicDecoratorConfig now